aten/src/ATen/test/scalar_test.cpp

```diff
@@ -128,3 +128,13 @@ TEST(TestScalar, TestScalar) {
   ASSERT_EQ(float_one.item<int32_t>(), 1);
   ASSERT_EQ(float_one.item<at::Half>(), 1);
 }
+
+TEST(TestScalar, TestConj) {
+  Scalar int_scalar = 257;
+  Scalar float_scalar = 3.0;
+  Scalar complex_scalar = c10::complex<double>(2, 3);
```

Comment:
nit: probably good to use non .0 values just to be sure you haven't accidentally done something like cast to int first.

Reply:
I am not sure I follow ...

Reply:
Greg is saying that if there was a a bug where the Scalar constructor for complex numbers accidentally represented the complex number as integers, you wouldn't catch it with this test. Need fractional value to see it.

Reply:
oh ok